### ISSUE TITLE
Usersnap Feedback - I was looking at the note from adrian.king about the footnotes not working[...]

### ISSUE THREAD

**Issue by ascott** [participedia/usersnaps#972]
**Sender**: pscully@clearviewconsultingllc.com
**Comment**: I was looking at the note from adrian.king about the footnotes not working, so I tested it in this entry. In this instance, clicking on the footnote simply opens the same page in another tab, instead of jumping the user to the footnote at the bottom of the page. Is there a way we can *simply* turn off all links from the footnotes in all entries?
[Open #666 in Usersnap Dashboard](https://usersnap.com/a/#/participedia-dev/p/participedia-net-9cb6d041/666)

<a href='https://usersnappublic.s3.amazonaws.com/2020-01-27/18-30/18c0ca4d-b7c8-4b72-bf36-9634941c482a.png'>![Usersnap Feedback - I was looking at the note from adrian.king about the footnotes not working(...)](https://usersnappublic.s3.amazonaws.com/2020-01-27/18-30/da192d30-6a77-4a7e-a135-4814a9d5e64c.jpg)</a>
<a href='https://usersnappublic.s3.amazonaws.com/2020-01-27/18-30/18c0ca4d-b7c8-4b72-bf36-9634941c482a.png'>Download original image</a>

**Browser**: Chrome 79 (Windows 10)
**Referer**: [https://participedia.net/method/5086](https://participedia.net/method/5086)
**Screen size**: 1920 x 1200 **Browser size**: 1920 x 1067

Powered by [usersnap.com](https://usersnap.com/?utm_source=github_entry&utm_medium=web&utm_campaign=product)

**Comment by jesicarson** [comment]
@plscully I don't think there's a way to turn off all footnote links exclusively, but @ascott would need to weigh in. 
Also, I thought we had replaced the old narrative editor with a new one that included footnotes? I'll check. 
If that's the case, I think it would mean that old entries will have broken footnotes, and I would assume that Scott and Jaskiran are working on this in the editing process? I will slack Jaskiran and ask.

**Comment by jesicarson** [comment]
Ok so we don't support footnotes at all, and Jaskiran either replaces broken links with a link to the actual source (navigating people away from the page) or notes them as broken links in the editing spreadsheet. This system seems flawed, and I think I agree with Pat that all footnote links should be removed - should I tell Jaskiran to do that from now on instead of linking them to external pages?

**Comment by plscully** [comment]
Yes, I think we should remove external links from footnotes. I am sorry if Scott at one time told me he was doing this and I somehow missed it. That approach doesn't make sense because (1) the convention for footnotes/endnotes is that they are telling you to look at the bottom of the page or end of the document, and (2) we don't want to encourage people to leave the site unless it is absolutely clear that this is what selecting something will do, as it is with our "links" field.

**Comment by jesicarson** [comment]
Agreed. I'll ask Jaskiran to update her editing protocol to simply remove all hyperlinks from in-text footnotes.
